src/GenFu/GenFu.cs

```diff
@@ -151,6 +151,8 @@ public class Defaults
             public static DateTime MIN_DATETIME = DateTime.Now.AddYears(-30);
             public static DateTime MAX_DATETIME = DateTime.Now.AddYears(30);
             
+            public static int MIN_SEED = 2;
```

Comment:
I like the idea of it but it seems odd to have two fields for it. How about a single one called something like PercentageOfNullablesFilled or something similar?